### DISCUSSION TITLE
Unable to publish Action to Marketplace

### DISCUSSION THREAD

**Discussion by akasprzok** [community/community#51893] [Actions]
### Select Topic Area

Question

### Body

I'm trying to publish github.com/massdriver-cloud/actions to the Actions MarketPlace, however I seem to be unable to do so. [Following the guide](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace), I draft a release but see neither "Release Actions" nor "Publish this Action to the GitHub Marketplace".

How can I debug why this might not be showing up?

* The repo is public
* the action.yml is in the root folder
* The name is unique afaik

Thanks!

**Comment by mnrendra** [comment]
I have the same problem. Could anyone help us? I can't see the "Publish this Action to the GitHub Marketplace" checkbox to publish my GitHub Actions source code. Thank you!

**Comment by jsoref** [comment]
It seems like you're on the marketplace: https://github.com/marketplace/actions/massdriver-actions

**Comment by akasprzok** [comment]
Apologies, I ended up reaching out to GH support. Apparently there was some sort of bug that prevented us from publishing the Action, which GH fixed on their side.

I do not have any details beyond that.

**Reply by neoblizz** [reply]
Through what method did you reach out to them?

**Reply by akasprzok** [reply]
A ticket.
